source/Sashimi/AzureWebAppHealthCheckActionHandler.cs

```diff
@@ -26,4 +26,25 @@ public IActionHandlerResult Execute(IActionHandlerContext context, ITaskLog task
                           .Execute(taskLog);
         }
     }
+
+    class AzureWebAppDiscoveryActionHandler : IActionHandler
```

Comment:
Might be worth moving this to it's own file to match the remainder of the code in the repo.

Reply:
Done.

Reply:
Done.

Reply:
Done.